airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/sponsored_display.py

```diff
@@ -12,12 +12,23 @@ class SponsoredDisplayCampaigns(SubProfilesStream):
     https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#/Campaigns
     """
 
+    def __init__(self, *args, **kwargs):
+        self.state_filter = kwargs.get("config", {}).get("state_filter")
+        super().__init__(*args, **kwargs)
```

Comment:
Can you move super() call in top of method?

Reply:
resolved